documents/motions/03.2025.1 - Objectives.md

```diff
@@ -26,7 +26,7 @@ The Objectives of the WCA are:
 3. Regulate
    1. To compile and enforce regulations governing Speedcubing and to ensure that such regulations are applied at all WCA Competitions.
    2. To create and enforce a mechanism whereby disputes and disciplinary matters within Speedcubing are subject to procedures set out in the applicable regulations and to bring all disputes to final resolution by arbitration.
-   3. To supervise and enforce the rights and duties of Staff, Registered Speedcubers, and Partners.
+   3. To supervise and enforce the rights and duties of the WCA Volunteers, Registered Speedcubers, and Partners.
```

Comment:
```suggestion
   3. To supervise and enforce the rights and duties of WCA Volunteers, Registered Speedcubers, and Partners.
```
Would this be better without a definite article? Other uses of the term omit the article, except one mention in the Dissolution motion where the article is retained from the existing wording, and has more contextual relevance when referring to Volunteers as a body.